### ISSUE TITLE
Chrome images for Selenoid are updated 

### ISSUE THREAD

**Issue by Zlaman** [aerokube/images#693]
Thanks for the great project.

Last Chrome image available is selenoid/chrome:128.0 https://hub.docker.com/r/selenoid/chrome/tags
Chrome 131 is already released. Will Chrome images for versions 129, 130, 131 and future versions be released?

**Comment by ajaysuryavamshi** [comment]
Same problem.
This is causing incompatibility issues with the latest selenium versions

**Comment by aandryashin** [comment]
Please consider this our project:
https://aerokube.com/moon

For selenoid you have to build images yourself:
https://aerokube.com/images/latest/

сб, 7 дек. 2024 г., 21:03 Ajay Suryavamshi ***@***.***>:

**Comment by jeremie-code** [comment]
Hi @aandryashin, 
So this is an official confirmation than repository https://hub.docker.com/u/selenoid won't get any more browser images? 
Thanks

**Comment by aandryashin** [comment]
What do you mean "official confirmation"? These repositories have
maintainer who built images, unfortunately he is passed away suddenly after
disease.

вс, 8 дек. 2024 г., 15:49 jeremie-code ***@***.***>:

**Comment by jeremie-code** [comment]
I’m really sorry for your loss, i couldn't think it was the reason, I can't imagine how hard it is.

**Comment by ajaysuryavamshi** [comment]
@aandryashin Sorry to hear this.
Can anyone help me or guide me on how to build the custom images with latest browser versions?

**Comment by aandryashin** [comment]
Here is build instruction:

https://aerokube.com/images/latest/#_building_procedure

Here is sources:

https://github.com/aerokube/images

пн, 9 дек. 2024 г., 11:55 Ajay Suryavamshi ***@***.***>:

**Comment by ajaysuryavamshi** [comment]
@aandryashin Thanks for the quick response.
Team, getting some weird exceptions building the images

Can anyone help me with the building the images from scratch in detail

**Comment by elvovsky** [comment]
@aandryashin Sasha, just to confirm what you said before about using the moon project. That project is to use Kubernetes instead of standalone docker containers for selenoid ?

**Comment by aandryashin** [comment]
Yes, moon is reliable, distributed system running on kubernetes cluster.
You can read more about it on:
https://blog.aerokube.com

ср, 11 дек. 2024 г., 17:26 elvovsky ***@***.***>:
